packages/core/src/aad.ts

```diff
@@ -233,21 +230,15 @@ export class AzureAD {
       : null;
   }
 
-  /**
-   * Gets a token silently, or falls back to interactive redirect.
-   */
-  private async getTokenRedirect(
-    silentRequest: SilentRequest,
-    interactiveRequest: RedirectRequest
-  ): Promise<AuthenticationResult | void> {
-    try {
-      return await this.msalApplication.acquireTokenSilent(silentRequest);
-    } catch (e) {
-      if (e instanceof InteractionRequiredAuthError) {
-        return this.msalApplication
-          .acquireTokenRedirect(interactiveRequest)
-          .catch(noop);
-      }
+  private setLocalAccountIdToLocalStorage(localId?: string): void {
+    if (!localId) {
+      localStorage.removeItem(accountLocalStorageKey);
+    } else {
+      localStorage.setItem(accountLocalStorageKey, localId);
```

Comment:
sdk used to work in node previously (not the authentication part, but it worked when if token is provided from outside),
do we want to make aad-auth work in node as well? or is it out of the current scope?

Reply:
you still can provide `accessToken` via `loginWithOAuth`, as it was before. The only question here that you can't refresh this token in any way in the node environment